hilfe/handbuch/einstellungen/benutzerkonto/index.md

```diff
@@ -13,7 +13,7 @@ Sofern ein Administrator Zugang zum "Benutzerkonto" hat, kann er in diesem Menü
 
 {% figure caption="Benutzerdaten & Passwort ändern" %}
```

Comment:
kannst du hiervor bitte noch einen Satz schreiben, ansonsten taucht das in der Preview auf
![Screenshot 2024-08-27 at 11 49 06](https://github.com/user-attachments/assets/2649e450-8a2f-47b9-a8ec-9588947a506f)